wire-ios-request-strategy/Sources/Request Strategies/Feature configurations/GetFeatureConfigsActionHandler.swift

```diff
@@ -139,7 +141,7 @@ extension GetFeatureConfigsActionHandler {
         let conversationGuestLinks: FeatureStatus
         let digitalSignatures: FeatureStatus
         let fileSharing: FeatureStatus
-        let mls: FeatureStatusWithConfig<Feature.MLS.Config>
+        let mls: FeatureStatusWithConfig<Feature.MLS.Config>?
```

Comment:
This change really should part of API versioning. Do you know what api version is the 3.5.3 backend?

Reply:
I do not have such info

Reply:
I can see that in swagger mls have asterisk (required field) on the endpoint since V2, so the field should be there anyway. But the key is not present while decoding.

Reply:
check on dina

Reply:
I was testing on this backend, is there an endpoint that returns api version?

Reply:
Ohh I see api-version endpoint :)

Reply:
you'll see the supported versions and the one you're currently on

Reply:
@lewandowskit93 What you need to do is to use the api version to parse the response of get /feature-configs differently.

When parsing the response, you can get the api version from the response object. 

Check the swagger docs for the difference versions to see what the differences are.

I would advise to create a specific payload model for each version, switch over the api version and decode the payload with the appropriate model.